openid-connect-server/src/test/java/org/mitre/oauth2/service/impl/TestDefaultSystemScopeService.java

```diff
@@ -184,6 +184,60 @@ public void scopesMatch() {
 
 		// extra scope (fail)
 		assertThat(service.scopesMatch(expected, actualBad), is(false));
+		
 	}
+	
+	@Test
+	public void getremoveRestrictedAndReservedScopes() {
+						
+		Set<SystemScope> restrictedAndReservedScopes = Sets.newHashSet(defaultDynScope1, defaultDynScope2, dynScope1);
```

Comment:
bad variable name: it looks here like you're making a set of just the restricted and removed scopes, where you're really making a list of unrestricted scopes